README.md

```diff
@@ -1,5 +1,5 @@
 # uptasticsearch
-
+[![Travis Build Status](https://img.shields.io/travis/jameslamb/doppel-cli.svg?label=travis&logo=travis&branch=master)](https://travis-ci.org/uptake/uptasticsearch)
```

Comment:
Thanks! Be careful...this link still has `jameslamb/doppel-cli` in it. Could you please replace that with `uptake/uptasticsearch`?

Also, could you please have this thing replace the "Build Status" link on line three (instead of adding a new line)? 

Thank you for contributing!